pallets/node-authorization/src/tests.rs

```diff
@@ -459,3 +459,19 @@ fn test_generate_peer_id_invalid_utf8() {
 	let invalid_node_id: NodeId = vec![0xFF, 0xFE, 0xFD];
 	assert_err!(NodeAuthorization::generate_peer_id(&invalid_node_id), Error::<Test>::InvalidUtf8);
 }
+
+// the test to check that the PeerId not too long.
+#[test]
```

Comment:
This needs a rework.
Test all methods in `node-authorization` which emits `PeerIdTooLong`.